redbot/core/events.py

```diff
@@ -99,10 +100,65 @@ async def on_ready():
                     "Outdated version! {} is available "
                     "but you're using {}".format(data["info"]["version"], red_version)
                 )
-                outdated_red_message = (
+                outdated_red_message = _(
                     "Your Red instance is out of date! {} is the current "
                     "version, however you are using {}!"
                 ).format(data["info"]["version"], red_version)
+                requires_python = data["info"]["requires_python"]
+                current_python = platform.python_version()
+                extra_update = _(
+                    "\n\nWhile the following should work in most scenarios as it is "
+                    "based on your current OS, environment, and Python version, "
+                    "**we highly recommend you to read the update docs at <{docs}> and "
+                    "make sure there is nothing else that "
+                    "needs to be done during the update.**"
+                ).format(docs="https://docs.discord.red/en/stable/update_red.html",)
+                if expected_version(current_python, requires_python):
+                    installed_extras = []
+                    for extra, reqs in red_pkg._dep_map:
```

Comment:
My initial suggestion was incorrect.
```suggestion
                    for extra, reqs in red_pkg._dep_map.items():
```